CollapseLauncher/XAMLs/MainApp/Pages/OOBE/OOBESelectGameBG.xaml.cs

```diff
@@ -90,9 +91,18 @@ internal static async Task<bool> TryLoadGameDetails(PresetConfig config = null)
 
                 _gameHomepageLink = config.ZoneURL;
                 _gameDescription = config.ZoneDescription;
+                
+                _gamePosterPath = config.GameType switch
+                                {
+                                    GameNameType.Honkai => Path.Combine(AppFolder,   @"Assets\Images\GamePoster\poster_honkai.png"),
+                                    GameNameType.Genshin => Path.Combine(AppFolder,  @"Assets\Images\GamePoster\poster_genshin.png"),
+                                    GameNameType.StarRail => Path.Combine(AppFolder, @"Assets\Images\GamePoster\poster_starrail.png"),
+                                    GameNameType.Zenless => Path.Combine(AppFolder,  @"Assets\Images\GamePoster\poster_zzz.png"),
+                                    _ => AppDefaultBG
+                                };
 
                 // TODO: Use FallbackCDNUtil to get the sprites
-                _gamePosterPath = await ImageLoaderHelper.GetCachedSpritesAsync(FallbackCDNUtil.TryGetAbsoluteToRelativeCDNURL(config.ZonePosterURL, "metadata/"), default);
+                //_gamePosterPath = await ImageLoaderHelper.GetCachedSpritesAsync(FallbackCDNUtil.TryGetAbsoluteToRelativeCDNURL(config.ZonePosterURL, "metadata/"), default);
```

Comment:
Remove if not in use, or uncomment if you commented it for testing

Reply:
Intentional as the code might be used again if the local loading method is removed or we want to use different posters for OOBE